nekoyume/Assets/_Scripts/UI/Widget/MobileShop.cs

```diff
@@ -52,6 +57,24 @@ public class MobileShop : Widget
 
         public static L10NSchema MOBILE_L10N_SCHEMA;
 
+        protected override void Awake()
+        {
+            base.Awake();
+
+            closeButton.onClick.AddListener(() =>
+            {
+                Close(true);
+                Game.Event.OnRoomEnter.Invoke(true);
+                AudioController.PlayClick();
+            });
```

Comment:
```suggestion
            {
                CloseWidget.Invoke();
                AudioController.PlayClick();
            });
```
closeButton에서 호출하는 Close도 인자가 false여야하지않을까요?

Reply:
요거 어디서 보고 가져온거라 저도 기준을 잘 모르겠는데..
Popup류 위젯은 보통 false로쓰고, popup이 아닌 위젯은 true로 쓰고 있는듯 해서 통일한다면 둘 다 true로 가는게 맞는거같네요

Reply:
해당 bool 인자는 Show나 Close 호출할 때 애니메이션을 스킵한다는 내용이라 특이한 경우(shortcut)가 아니고서는 false가 맞습니다. 큰 이유가 없으면 false로 하는게 맞을 것 같아요.

Reply:
3b46f35d59529c5bc73457ea084cb2bd31e8d51e 에서 일괄 false로 변경했습니다